types/postcss-prefix-selector/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for postcss-prefix-selector 1.14
+// Project: https://github.com/RadValentin/postcss-prefix-selector
+// Definitions by: robertmaier <https://github.com/robertmaier>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/types/postcss-prefix-selector
+
+interface Options {
+    prefix?: string;
```

Comment:
isn't the prefix required?
https://github.com/RadValentin/postcss-prefix-selector/blob/master/index.js#L2
(and by the very purpose `Prefix every CSS selector with a custom namespace .a => .prefix .a`)